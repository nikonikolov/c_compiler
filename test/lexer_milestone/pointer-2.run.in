
int main()
{

  int x=0;

  int *y=&x;

  x=y[0];

}
