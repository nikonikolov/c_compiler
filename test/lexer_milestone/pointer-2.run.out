int Keyword Tint 2  2
main Identifier TIdentifier 2  2
( Operator T( 2  2
) Operator T) 2  2
{ Operator T{ 3  3
int Keyword Tint 5  5
x Identifier TIdentifier 5  5
= Operator T= 5  5
0 Constant TIntConstant 5  5
; Operator T; 5  5
int Keyword Tint 7  7
* Operator T* 7  7
y Identifier TIdentifier 7  7
= Operator T= 7  7
& Operator T& 7  7
x Identifier TIdentifier 7  7
; Operator T; 7  7
x Identifier TIdentifier 9  9
= Operator T= 9  9
y Identifier TIdentifier 9  9
[ Operator T[ 9  9
0 Constant TIntConstant 9  9
] Operator T] 9  9
; Operator T; 9  9
} Operator T} 11  11
